### DISCUSSION TITLE
Modbus address for pre-Nexus

### DISCUSSION THREAD

**Discussion by skipfire** [jgyates/genmon#1172] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

Appendix D has a mention of pre-Nexus controllers from around 2008 and that they need a Modbus address set, but it does not mention the address.  I found issue #828 that had one comment that stated 41, is this always the right value for these old ones?  If so, can Appendix D get updated with that number?  If it isn't always 41, how does one find the right address?

**Comment by jgyates** [comment, marked as answer]
I updated the wiki. Thanks!
